spinalcordtoolbox/scripts/sct_deepseg_sc.py

```diff
@@ -165,6 +169,11 @@ def main():
     else:
         manual_centerline_fname = None
 
+    if args.o is not None:
+        fname_out = args.o
+    else:
+        fname_out = extract_fname(fname_image)[1] + '_seg' + extract_fname(fname_image)[2]
```

Comment:
there is probably a more elegant way than launching the same API twice (extract_fname), maybe @joshuacwnewton you have some ideas?

Reply:
We can run it only once and to get the all the output and then use that.